irohad/synchronizer/synchronizer.hpp

```diff
@@ -38,11 +39,17 @@ namespace iroha {
           const shared_model::interface::BlockVariant &commit_message) = 0;
 
       /**
-       * Emit committed blocks
-       * Note 1: from one block received on consensus
-       * Note 2: commit may be empty; in that case, it can contain empty pointer
+       * Emit block(s) after commit event:
```

Comment:
after commit event -> after collaboration outcome.
As for description below, I think it can be simplified to something like "Zero or more blocks synchronized and outcome type".